tempodb/tempodb.go

```diff
@@ -253,21 +252,26 @@ func (rw *readerWriter) Find(ctx context.Context, tenantID string, id encoding.I
 			copiedBlocklist = append(copiedBlocklist, b)
 		}
 	}
+	numTenants := len(rw.blockLists)
 	rw.blockListsMtx.Unlock()
 
-	if !found {
+	if numTenants == 0 {
```

Comment:
let's leave this out for now.  this is a different issue and @mdisibio is going to be looking at it.

Reply:
This should fix https://github.com/grafana/tempo/issues/94 but I'll defer to @mdisibio for the fix!